src/services/validate-event.ts

```diff
@@ -54,17 +55,15 @@ export function validateLevelOfConfidence(intervention: EventsEnum, event: TxMAI
 
 /**
  * A function to validate that the event received is not in the future
- * @param eventEnum - the event name as an EventsEnum
  * @param event - the event received
  * @throws ValidationError - if the timestamp of the event is in the future
  */
-export async function validateEventIsNotInFuture(eventEnum: EventsEnum, event: TxMAIngressEvent) {
+export async function validateEventIsNotInFuture(event: TxMAIngressEvent) {
   const eventTimestampInMs = event.event_timestamp_ms ?? event.timestamp * 1000;
   const now = getCurrentTimestamp().milliseconds;
   if (now < eventTimestampInMs) {
     logger.debug(`Timestamp is in the future (sec): ${eventTimestampInMs}.`);
     logAndPublishMetric(MetricNames.INTERVENTION_IGNORED_IN_FUTURE);
-    await sendAuditEvent('AIS_EVENT_IGNORED_IN_FUTURE', eventEnum, event);
```

Comment:
why has this been deleted? My understanding is that we are still wanting to publish this type of event to TxMA we just won't be able to provide the information about the user's current state, allowable interventions etc